### ISSUE TITLE
Enabling/disabling do not disturb schedule causes pending changes to not be saved on exit

### ISSUE THREAD

**Issue by GabrielDTB** [GrapheneOS/os-issue-tracker#3158]
After editing the details of a do not disturb schedule, if the enabled state is toggled before exiting back to the screen that lists all of the schedules, the changes to said schedule are not saved.

**Comment by ogghi** [comment]
https://discuss.grapheneos.org/d/14447-dnd-not-keeing-the-setting/3
Is this the same as in my description on the forum here?

**Comment by GabrielDTB** [comment]
@ogghi I don't think so. This bug is about manually toggling enabled while editing the settings.

**Comment by ogghi** [comment]
OK, then I'll check and open another bug in case

**Comment by GabrielDTB** [comment]
Modes seems to not have this issue.
